src/FlatButton/FlatButton.js

```diff
@@ -36,6 +36,10 @@ class FlatButton extends Component {
      */
     className: PropTypes.string,
     /**
+     * Disables the ripple effect.
```

Comment:
I think that we can follow the IconButton wording:
`* If true, the element's ripple effect will be disabled.`